datafusion/expr/src/function.rs

```diff
@@ -296,15 +296,15 @@ pub fn signature(fun: &BuiltinScalarFunction) -> Signature {
         | BuiltinScalarFunction::SHA256
         | BuiltinScalarFunction::SHA384
         | BuiltinScalarFunction::SHA512
-        | BuiltinScalarFunction::Trim
         | BuiltinScalarFunction::Upper => Signature::uniform(
             1,
             vec![DataType::Utf8, DataType::LargeUtf8],
             fun.volatility(),
         ),
         BuiltinScalarFunction::Btrim
         | BuiltinScalarFunction::Ltrim
-        | BuiltinScalarFunction::Rtrim => Signature::one_of(
```

Comment:
Given that trim originally supported LargeUtf8 and l/r/btrim might also support those types, does it make sense to update the function signature here to accept largeUtf8's as well?

Reply:
Tests are passing, which must mean that we did not have any tests for `Trim` with `LargeUtf8` so I think this change is fine as is. If we declare that `LargeUtf8` is supported then we need to add relevant tests as well.